index.css

```diff
@@ -0,0 +1,218 @@
+body {
+    background-color:#1b1b2f;
+    width: auto;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+/* navigation */
```

Comment:
What led to the decision to put a comment here describing the following code, rather than moving this code to a separate style sheet (e.g. navigation.css)?  The feel that you needed a comment here was indication you could improve the cohesion of your code by applying the Single Responsibility Principle, and moving this to it's own file.